src/markdown-pages/docs/ecosystem/ecosystem-manage-cps.md

```diff
@@ -79,41 +92,37 @@ For a complete list of supported parameters see the <a href="https://github.com/
 Properties are returned in summary format. For example:
 
 ```
-namespace name                           value
-docker    docker.engine.LOCAL.hostname   127.0.0.1
-docker    docker.engine.REMOTE.hostname  103.67.89.6
+namespace name                    value
+docker    engine.LOCAL.hostname   127.0.0.1
+docker    engine.REMOTE.hostname  103.67.89.6
 
 Total: 2
 ```
 
 If the property value that is returned is longer than 50 characters, the first fifty characters are displayed, followed by three ellipses (`...`). For example:
 
 ```
-namespace name         value
-framework test.stream  this-value-is-60-characters-long-xxxxxxxxxxxxxxxxx...
+namespace name              value
+framework credentials.store etcd:http://this-value-is-60-characters-long-xxxxx... 
 
 Total: 1
 ```
 
 
-Returned properties are first sorted according to the number of segments in the property name (denoted by the period character (.)), and then in alphabetical order. For example:
-
-```
-namespace name                          value
-framework framework.test.stream.a.b     account1
-framework framework.test.stream.b.c     account2
-framework framework.test.stream.a       myteststream
-framework framework.test.stream         bestsofar
-
-Total: 4
-```
+Returned properties are sorted in alphabetical order. 
```

Comment:
That's not true. They are in alphabetical order, but the first elements are those which have the most number of segments.

Reply:
I was looking at some output and it didn't look like it was done on segments which is why I changed it but then I looked again and I think I misinterpreted the return results ordering. Have added back in.